Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -489,6 +497,10 @@ void ProgramShaderCache::Shutdown()
 			u8 *binary = data + sizeof(GLenum);
 			GLenum *prog_format = (GLenum*)data;
 			glGetProgramBinary(entry.second.shader.glprogid, binary_size, nullptr, prog_format, binary);
+			if (glGetError() != GL_NO_ERROR)
```

Comment:
I think I'll initialize all error-checking variables to their invalid state so I can detect that an error occurred without relying on `glGetError`.